### DISCUSSION TITLE
conversion of ml/min/ 10e6 Cell  to ml/min/kg

### DISCUSSION THREAD

**Discussion by Abderaoufradjai** [Open-Systems-Pharmacology/Forum#891] [Questions & Problems] [answered]
I have simulated human hepatic clearance value in OPERA with the following unit (0.17ml / min / 10e6 Cell), 
Can you please help me to convert this unit to (ml / min / kg) so that I can use in PKsim.

many thanks.

**Comment by tobiasK2001** [comment, marked as answer]
Hi,

that is unfortunately not directly possible yet. But please check out the threads here, [https://github.com/Open-Systems-Pharmacology/Forum/issues/306](url) and here[https://github.com/Open-Systems-Pharmacology/Forum/issues/177](url) to find some inspiration.

hope that might help you,

Best Tobias

**Comment by Abderaoufradjai** [comment]
Hi,
Thank you for your answer,
I have another question,
What do you think about the use of the input parameters values generated from the QSAR methods(OPERA, ADMETlab....) in PKsim and the reliability of these results ( Pk-analysis ) 

Many thanks,
Abderaouf

**Comment by Abderaoufradjai** [comment]
And can I validate my model without having observed data ?

Thank you again,
Abderaouf.

**Comment by tobiasK2001** [comment]
well regarding 

> 

> use of the input parameters values generated from the QSAR methods(OPERA, ADMETlab....) 
They may be useful as starting point, but most often they have to be fitted against observed data.
This answers your second question as well.

Best, Tobias
